build.yaml

```diff
@@ -4789,20 +4780,6 @@ targets:
   - mac
   - linux
   - posix
-- name: lb_load_data_store_test
```

Comment:
@nicolasnoble, do I understand correctly that we don't currently run tests via bazel, only via make?  If so, wouldn't this prevent these tests from running automatically?

Reply:
@nicolasnoble Should I keep these targets and close this PR?

Reply:
I thought I replied to this one before, but apparently, I didn't. Sorry.

So, two things:

1) Bazel tests are running - people aren't necessarily paying attention to them, even though I keep flaying my arms to try and tell people to use Bazel more and more :)

Also, they are not really aptly named at the moment:

![image](https://user-images.githubusercontent.com/7281574/41686835-66513416-749a-11e8-90c7-631cd21af7a9.png)

2) There is no way to make this work using the Makefile, really, so we'll have to make do with Bazel only.

The main lesson here is that people need to really start paying attention to these checkboxes.

Reply:
Thanks! Since the Bazel tests are working properly, we can remove these targets here safely. 

Although some parts of the load reporting feature don't rely on OpenCenus and can be tested with Make, we may still want to remove them to avoid inconsistency and confusion.

@markdroth I will clean this PR soon.